### DISCUSSION TITLE
Tutorial on style variations in theme.json

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#103] [Topic Ideas]
This post would be a full tutorial on how theme developers can add styles for custom style variations via `theme.json`. It should also cover how this applies to the user experience in the Styles interface. And provide examples of what is possible and any potential issues.

**Resources:**

- [Documentation](https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-json/#variations)
- https://github.com/WordPress/gutenberg/pull/46343

**Comment by bph** [comment]
Created an issue so this can move forward. Please continue discussion there. 

- https://github.com/WordPress/developer-blog-content/issues/113

This discussion is locked.

**Comment by bph** [comment]
Published: [Customizing core block style variations via theme.json](https://developer.wordpress.org/news/2023/05/customizing-core-block-style-variations-via-theme-json/)
